src/ResourceParser.h

```diff
@@ -49,21 +52,15 @@ namespace snowcrash {
 
                 // Make this section an action
                 if (!captureGroups[2].empty()) {
+                    return processNestedAction(node, siblings, pd, layout, out);
```

Comment:
I thought that it would yield the same result, and since all the tests passed I figured I was right... Want me to revert?